packages/editor-ui/src/components/Node/NodeCreator/CategorizedItems.vue

```diff
@@ -78,6 +78,8 @@
 <script lang="ts">
 import Vue, { PropType } from 'vue';
 import camelcase from 'lodash.camelcase';
+import { intersection } from 'lodash';
```

Comment:
maybe instead of using `lodash` you could
```ts
const intersection = <T>(a: T[], b:T[]): T[] => {
    const setA = new Set(a);
    return b.filter(setA.has);
}
```